pages/quantum.js

```diff
@@ -53,139 +61,453 @@ import {
+          <h4 class="TextFormattingH4">Social Button</h4>
+          <h6 class="TextFormattingH6">
+            These buttons are used to run actions related to their respective
+            providers. Example: login with OAuth API from Google.
+          </h6>
+          <SocialButton provider="facebook" />
+        </div>
+        <div class="Width  ComponentsContainer WhiteBackground RoundedEdges">
+          <h4 class="TextFormattingH4">Info Icon</h4>
+          <Icon name="info" />
+        </div>
+        <div class="Width  ComponentsContainer WhiteBackground RoundedEdges">
+          <h4 class="TextFormattingH4">Avatar</h4>
+          <h6 class="TextFormattingH6">
+            Avatars are used to display a thumbnail of the user's profile
+            picture or an user icon if the picture is not defined. It can also
+            display a dot to alert that there is some notification.
+          </h6>
+          <Avatar />
+        </div>
+        <div class="Width   ComponentsContainer WhiteBackground RoundedEdges">
+          <h4 class="TextFormattingH4">Socials</h4>
+          <h6 class="TextFormattingH6">
+            Socials show the icons of respective Social Medias{" "}
+          </h6>
+          <Socials items={socialMocks} />
+        </div>
+      </div>
+      <div class="FlexContainer ColoredBackground PaddingLeftRight MarginTopBottom Overflow ">
+        <div class="Width  ComponentsContainer WhiteBackground RoundedEdges">
+          <h4 class="TextFormattingH4">Checkbox Group</h4>
+          <h6 class="TextFormattingH6">
+            Gets values from a single onChange callback prop and apply a single
+            error message to a group of checkboxes.
+          </h6>
+          <CheckboxGroup
+            options={[{ checked: false, label: "Test", name: "Test" }]}
+          />
+        </div>
+        <div class="Width  ComponentsContainer WhiteBackground RoundedEdges">
+          <h4 class="TextFormattingH4">Radio Group</h4>
+          <h6 class="TextFormattingH6">
+            Radio group is a list of radio buttons that are used when a list of
+            two or more options are mutually exclusive, meaning the user must
+            select only one option.
+          </h6>
+          <RadioGroup
+            name="test"
+            options={[
+              {
+                value: "Test",
+                label: "Test",
+              },
+            ]}
+          />
+        </div>
+        <div class="Width   WhiteBackground RoundedEdges ComponentsContainer">
+          <h4 class="TextFormattingH4">Toggle</h4>
+          <h6 class="TextFormattingH6">
+            Toggle is a control that is used to quickly switch between two
+            possible states.
+          </h6>
+          <Toggle id="right" />
+        </div>
+        <div class="Width   WhiteBackground RoundedEdges ComponentsContainer">
+          <h4 class="TextFormattingH4">Form</h4>
```

Comment:
how about of using form items from quantum here?
Using the Input component with its labels and helper text?